gsdk/scale-gen/src/main.rs

```diff
@@ -0,0 +1,101 @@
+// This file is part of Gear.
```

Comment:
I'd consider appending dump of .scale into vara-runtime build.rs

Reply:
Good idea!